book.rb

```diff
@@ -0,0 +1,15 @@
+require './rental'
+
+class Book
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
+
+  attr_accessor :title, :author, :rentals
+
+  def add_rental(date, person)
+    Rental.new(date, book, person)
+  end
```

Comment:
- [ ] I suggest that you use the `self` in place of `book` keyword to refer to the current class itself within the scope of that class only whereas